docs/deprecations/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Deprecations
+description: Upcoming and past deprecations by version for Octopus Server
+position: 300
+---
+
+## Overview
+
+From time to time, Octopus will deprecate features that are no longer going to be supported, and will eventually be removed.
+
+Deprecations have the following lifecycle:
+
+- Announce deprecation
+- (+6 months) Toggle off deprecated functionality
+- (+1 year) Remove deprecated functionality
+
+:::warning
+Deprecations are subject to change in detail or timeframe. If you need help assessing the impact of a deprecation on your particular Octopus Server configuration, please please [email our support team](mailto:support@octopus.com)
+:::
+
+## Deprecations for 2023.1
+
+None.
```

Comment:
Minor point: Is this a placeholder? Or would every version have a heading - that could get large quickly (4+ a year)? Would it be better to place a deprecation header when we deprecate things in a specific version?

Reply:
It is a placeholder. And yes we will likely only put a header in if we are deprecating something. I more just wanted to communicate the structure of it for now.